src/Azure/Orleans.Streaming.AzureStorage/Providers/Streams/PersistentStreams/StreamDeliveryFailureEntity.cs

```diff
@@ -23,7 +23,7 @@ public class StreamDeliveryFailureEntity : TableEntity
         /// <summary>
         /// Guid Id of the stream on which the failure occurred.
         /// </summary>
-        public Guid StreamGuid { get; set; }
+        public string StreamGuid { get; set; }
```

Comment:
I guess we should rename this in a subsequent PR

Reply:
We could, but it was to keep some BC for cheap if possible. Not sure it is worth it